.github/ISSUE_TEMPLATE/2-feature-request.md

```diff
@@ -1,79 +1,62 @@
 ---
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
 name: Feature request
-
 about: Suggest how to improve Traffic Control with something new
-
 labels: enhancement
-
 ---
-
 <!--
 ************ STOP!! ************
 If this issue identifies a security vulnerability, DO NOT submit it! Instead, contact
 the Apache Traffic Control Security Team at security@trafficcontrol.apache.org and follow the
-guidelines at https://www.apache.org/security/ regarding vulnerability disclosure.
+guidelines at https://apache.org/security regarding vulnerability disclosure.
 
 - For *SUPPORT QUESTIONS*, use the #traffic-control channel on the ASF slack (https://s.apache.org/slack-invite)
-or Traffic Control mailing lists (https://trafficcontrol.apache.org/mailing_lists).
-- Before submitting, please **SEARCH GITHUB** for a similar issue or PR.
+or the Traffic Control Users mailing list (send an email to users-subscribe@trafficcontrol.apache.org to subscribe).
+- Before submitting, please **SEARCH GITHUB** for a similar issue or PR
+    * https://github.com/apache/trafficcontrol/issues
+    * https://github.com/apache/trafficcontrol/pulls
 -->
 
 ## I'm submitting a ...
-<!-- delete all those that don't apply -->
-<!--- security vulnerability (STOP!! - see above)-->
--  new feature / enhancement request
--  other <!--(Please do not submit support requests here - see above)-->
+<!-- Do not submit a security vulnerability here - see above security vulnerability info -->
+- [x] new feature / enhancement request
+<!-- Please do not submit support requests here - see above-->
 
 ## Traffic Control components affected ...
 <!-- delete all those that don't apply -->
--  CDN in a Box
--  Documentation
--  Grove
--  Traffic Control Client
--  Traffic Monitor
--  Traffic Ops
--  Traffic Ops ORT
--  Traffic Portal
--  Traffic Router
--  Traffic Stats
--  Traffic Vault
--  CI tests
--  unknown
+- Traffic Control Cache Config (T3C)
+- Traffic Control Client <!-- Please specify which (Python, Go, or Java) -->
+- Traffic Monitor
+- Traffic Ops
+- Traffic Portal
+- Traffic Router
+- Traffic Stats
+- Grove
+- Documentation
+- CDN in a Box
+- Automation (GitHub Actions, Ansible Roles)
+- unknown
 
 ## Current behavior:
```

Comment:
seems like `Current Behavior` could be removed from the feature request template to make it even easier...

Reply:
Yeah, maybe combine it into this?
```
## Description
<!-- Describe how the current ATC features are insufficient and how this new feature would improve Traffic Control -->
```

Reply:
I guess we should probably do something similar for the improvement request and bug report?

Now that I type that out, do we even need to differentiate between improvement and feature requests? Those two have always been kind of confusing. Maybe we should get rid of one altogether?

Reply:
The lines between those are blurry, IMO. For similar reasons I'm also not sure we need both the `enhancement` and `improvement` labels, but that's a separate discussion.

Reply:
> seems like `Current Behavior` could be removed from the feature request template to make it even easier...

> Yeah, maybe combine it into this?

Combined in b6b04bee11.

> I guess we should probably do something similar for the improvement request and bug report?

I don't think that we should combine *Current behavior* and *Expected behavior* in the bug report, although maybe we could reorder them. The [Go issue template](https://github.com/golang/go/blob/9eee0ed439/.github/ISSUE_TEMPLATE#L24-L38) asks
* What did you do?
* What did you expect to see?
* What did you see instead?

Asking those 3 questions is a good model for bug reports IMO.

> Now that I type that out, do we even need to differentiate between improvement and feature requests? Those two have always been kind of confusing. Maybe we should get rid of one altogether?

> The lines between those are blurry, IMO.

If *improvement* and *new feature* seem similar, we should definitely reword it in the templates to make the difference clearer.
Pushed changes to the *Improvement request* template:
* Replaced *feature* with *change* in 54c7df03f2
* Removed the *Minimal reproduction of the problem with instructions* section in 9e09a8ca92
*   Changed the description on https://github.com/apache/trafficcontrol/issues/new/choose to
    > Suggest how to improve an existing feature of Traffic Control (usability, performance, tech debt, etc.)

    in d1b3d70726

Reply:
> I'm also not sure we need both the `enhancement` and `improvement` labels, but that's a separate discussion.

What about renaming the *enhancement* label to *new feature*?

Reply:
that would certainly help.

Reply:
Renamed to *new feature* in 05fa1a7434

Reply:
i think the diff between improvement and new feature was supposed to be:

improvement: you can do this already but it's really painful so it needs improved
new feature: you can't do this at all

but i have no problem combining them. in the issue template and the labels.

Reply:
> but i have no problem combining them. in the issue template and the labels.

Do they need to be combined, now that *enhancement* has been renamed to *new feature*?